src/PKSim.Infrastructure/ProjectConverter/v7_2/Converter710To720.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System.Xml.Linq;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Services;
+using OSPSuite.Utility.Visitor;
+using PKSim.Core;
+using PKSim.Core.Model;
+using PKSim.Core.Repositories;
+using PKSim.Core.Services;
+
+namespace PKSim.Infrastructure.ProjectConverter.v7_2
+{
+   public class Converter710To720 : IObjectConverter,
+      IVisitor<Individual>,
+      IVisitor<Population>,
+      IVisitor<IndividualSimulation>,
+      IVisitor<PopulationSimulation>
+
+   {
+      private readonly IDefaultIndividualRetriever _defaultIndividualRetriever;
+      private readonly ICloner _cloner;
+      private readonly ICalculationMethodRepository _calculationMethodRepository;
+      private readonly Individual _defaultHuman;
+      private readonly IEntityPathResolver _entityPathResolver;
+
+      public Converter710To720(IDefaultIndividualRetriever defaultIndividualRetriever, ICloner cloner, ICalculationMethodRepository calculationMethodRepository, IEntityPathResolver entityPathResolver)
+      {
+         _defaultIndividualRetriever = defaultIndividualRetriever;
+         _cloner = cloner;
+         _calculationMethodRepository = calculationMethodRepository;
+         _entityPathResolver = entityPathResolver;
+         //use own instance so that we can manipulate parameter to calculate BSA for population
+         _defaultHuman = _cloner.Clone(defaultIndividualRetriever.DefaultHuman());
+      }
+
+      public bool IsSatisfiedBy(int version)
+      {
+         return version == ProjectVersions.V7_1_0;
+      }
+
+      public int Convert(object objectToConvert, int originalVersion)
+      {
+         this.Visit(objectToConvert);
+         return ProjectVersions.V7_2_0;
+      }
+
+      public int ConvertXml(XElement element, int originalVersion)
+      {
+         return ProjectVersions.V7_2_0;
+      }
+
+      public void Visit(Individual individual)
+      {
+         convertIndividual(individual);
+      }
+
+      public void Visit(Population population)
+      {
+         convertIndividual(population.FirstIndividual);
+         addBSAParameterValues(population);
+      }
+
+      private void convertIndividual(Individual individual)
+      {
```

Comment:
I am only converting the Individual and Population. (building block and in the simulation). I am not touching the simulations